pkg/util/metrics_helper_test.go

```diff
@@ -508,6 +510,463 @@ func TestFloat64PrecisionStability(t *testing.T) {
 	}
 }
 
+func TestMetrics(t *testing.T) {
```

Comment:
What is this test about? Is it really worth to keep it? If yes, I would suggest to rename it to make it more clear to understand.

Reply:
This is a baseline for other tests, for comparison. I'd keep it, I'll try to rename it with other tests.